assets/docs/callmemehdi/models/AraBERT/1.md

```diff
@@ -1,6 +1,6 @@
 # Module callmemehdi/arabert/1
```

Comment:
Please update the 'arabert' casing here or in the folder name so that they match